components/kyma-environment-broker/internal/dto.go

```diff
@@ -27,7 +27,15 @@ func (p ProvisioningParameters) IsEqual(input ProvisioningParameters) bool {
 	if !reflect.DeepEqual(p.ErsContext, input.ErsContext) {
 		return false
 	}
-	if !reflect.DeepEqual(p.Parameters, input.Parameters) {
+
+	// TODO: If there are already not resolved operations in the db
+	//       KEB will raise an error "provisioning operation already
+	//       exist" after updating the image on the environments.
+	//       Revert this check after some time, when every operation
+	//       has at least the default Gardener Shoot Purpose to:
+	//       if !reflect.DeepEqual(p.Parameters, input.Parameters) {
```

Comment:
We should add a follow-up task for that. Do we have one?

Reply:
Sure! I'll create one :D I wanted to hear the Gopher's opinion on this solution first

Reply:
should not be changed, discussed